src/kili/services/export/format/kili/__init__.py

```diff
@@ -2,14 +2,25 @@
 
 import json
 from pathlib import Path
-from typing import Callable, Dict, List
+from typing import Callable, Dict, List, Optional, Union
 
 from kili.services.export.exceptions import NotCompatibleInputType, NotCompatibleOptions
-from kili.services.export.format.base import AbstractExporter
+from kili.services.export.format.base import AbstractExporter, reverse_rotation_vertices
 from kili.services.export.media.video import cut_video
 from kili.services.types import Job
 
 
+def _clean_json_response(asset: Dict):
+    if asset.get("latestLabel", {}) and asset["latestLabel"].get("jsonResponse", {}):
+        if "ROTATION_JOB" in asset["latestLabel"]["jsonResponse"]:
+            asset["latestLabel"]["jsonResponse"].pop("ROTATION_JOB")
+    if asset.get("labels"):
+        for label in asset["labels"]:
+            if label.get("jsonResponse", {}) and "ROTATION_JOB" in label["jsonResponse"]:
```

Comment:
Why are you mixing 2 syntaxes to do the same thing ?
`"key" in asset` or `asset.get("key", {})`
Use `in` to check if a key is in a dict.

Reply:
No I mean
if "key" in asset:
Instead of
fi asset.get("key", {}):